### DISCUSSION TITLE
Set max-build-height per world

### DISCUSSION THREAD

**Discussion by Pantera07** [PurpurMC/Purpur#675] [__Dismissed__] [answered]
I suggest this.
Ability to configure max-build-height and min-build-height per world in purpur config
This can prevent build on nether roof.
Also, in the upcoming 1.18, the height of the world will increase up and down.

**Comment by craftish37** [comment]
(Extremely basic datapacks)

**Reply by Pantera07** [reply]
It modifies the world, not prevents builds.
It causes errors in worlds that have already been created.

**Comment by craftish37** [comment]
No, no and… no 🤣
That’s pretty wrong, it perfectly does what it’s made for 👀

**Comment by katekerllenevich** [comment, marked as answer]
This is a feature in datapacks
